assets/src/blocks/TableOfContents/TableOfContentsBlock.js

```diff
@@ -1,31 +1,30 @@
-import {SubmenuEditor} from './SubmenuEditor.js';
-import {example} from './example';
-import {getStyleLabel} from '../../functions/getStyleLabel';
+import {TableOfContentsEditor} from './TableOfContentsEditor.js';
+import {example} from './example.js';
+import {getStyleLabel} from '../../functions/getStyleLabel.js';
+import {BLOCK_NAME, P4_BLOCK_PREFIX} from '../Constants/Constants.js';
 
 const {__} = wp.i18n;
 
-const BLOCK_NAME = 'planet4-blocks/submenu';
-
-export const registerSubmenuBlock = () => {
+export const registerTableOfContentsBlock = () => {
   const {registerBlockType, getBlockTypes} = wp.blocks;
 
-  const blockAlreadyExists = getBlockTypes().find(block => block.name === BLOCK_NAME);
+  const blockAlreadyExists = getBlockTypes().find(block => block.name === BLOCK_NAME.tableOfContents.name);
```

Comment:
This should no longer be needed if we remove the block from the plugin 🤔

Reply:
Removed!